src/components/Books.css

```diff
@@ -0,0 +1,5 @@
+.container {
+  background-color: #f5f6fa;
+  padding-top: 20px;
+  padding-bottom: 30px;
+}
```

Comment:
- [ ] The styling has to be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). Please As requested by the previous reviewer, 👇🏻 Kindly address this 💯 👍🏻 The book should have 100% width of it's container as shown the original design. 

> - [ ] In terms of the styling, it could be improved too 👷🏼.  Please consider using all the available space moving the progress information all the way to the right! 🤓 🔧 


|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/215219495-a9c80e23-7fb6-415d-9a77-6f785962ad91.png) | ![im2](https://user-images.githubusercontent.com/79058364/215232637-61437dc5-7a9b-45cb-b895-7c7edf54bf7b.png)

_Originally posted by @anagudelogu in https://github.com/OsvaldoBC/bookstore-ob/pull/5#discussion_r1089518704_